cpp/daal/src/algorithms/dtrees/forest/oneapi/df_tree_level_build_helper_oneapi.i

```diff
@@ -507,7 +539,9 @@ services::Status TreeLevelBuildHelperOneAPI<algorithmFPType>::doLevelPartition(c
         DAAL_CHECK_STATUS_VAR(status);
     }
 
-    DAAL_CHECK_STATUS_VAR(partitionCopy(treeOrderBuf, treeOrder, 0, nRows));
+    UniversalBuffer _temp = treeOrder;
+    treeOrder             = treeOrderBuf;
+    treeOrderBuf          = _temp;
```

Comment:
Please use implemented [this](https://github.com/oneapi-src/oneDAL/blob/3310c7a9a41a2f0eb72082115c0db98484e4d58d/cpp/daal/src/services/service_utils.h#L56) functionality or at least close it into `{}`

Reply:
thanks, done